packages/scaffold-config/test/unit/supportFile.spec.ts

```diff
@@ -220,4 +220,88 @@ describe('supportFileComponent', () => {
       })
     }
   })
+
+  context('svelte', () => {
+    for (const mountModule of ['cypress/svelte'] as const) {
+      it(`handles ${mountModule} and JS`, () => {
```

Comment:
Any reason to use a `for` loop for an array with a single value - do we expect to add more to this array?